client/src/components/Panel/Panel.tsx

```diff
@@ -1,17 +1,29 @@
 import classNames from "classnames";
+
 import React from "react";
 
 import { create_text_maker } from "src/models/text";
 
 import { textLightColor } from "src/style_constants/index";
 
 import text from "./Panel.yaml";
+
 import "./Panel.scss";
 
 const text_maker = create_text_maker(text);
 
-export class Panel extends React.Component {
-  constructor(props) {
+type PanelProps = {
+  allowOverflow: boolean;
+  title: string;
+  otherHeaderContent: JSX.Element;
+  children: React.ReactNode;
+};
+
+interface isOpenInterface {
```

Comment:
Makes sense, got it.

Reply:
Also- by the way the branch has already been rebased in case you saw me mention rebasing in the PR comment, just forgot to remove it after I rebased on Monday.